app/src/main/java/com/github/se/eventradar/model/message/MessageHistory.kt

```diff
@@ -0,0 +1,36 @@
+package com.github.se.eventradar.model.message
+
+data class MessageHistory(
+    val fromUser: String,
```

Comment:
clarification: does this mean messages between two users are stored twice?

Reply:
No, they are interchangeable. The naming was more of a convention. I wanted to avoid "user1" and "user2". Open to suggestions.

Reply:
Hmm, I can only think of user1 and user2 to be honest, not a priority to be changed. Perhaps just add a comment in to avoid confusion.